File: core/src/main/java/hello/core/member/data/Member.java
package hello.core.member.data;

public class Member {

    /*
    * 요구 사항에 있는 것은
    * id
    * name
    * grade
    *
    * 위 3개 말고는 없음.
    * */


    private long id;
    private String name;
    private Grade grade;

    // 생성자 생성
    public Member(long id, String name, Grade grade) {
        this.id = id;
        this.name = name;
        this.grade = grade;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Grade getGrade() {
        return grade;
    }

    public void setGrade(Grade grade) {
        this.grade = grade;
    }
}

File: 김영한 Spring/hello-spring-kotlin/src/main/kotlin/kotlinhellospring/demo/domain/Member.kt
package kotlinhellospring.demo.domain

data class Member(
    var name: String? = null
) {
    /**
     * () 안에는 필수 생성자를 의미
     * 따로 뺴고 싶으면 {} 안에서 작성해야 함.
     */
    var id: Long? = null
}

